packages/test-commons/src/generateIsomorphicTests.ts

```diff
@@ -55,6 +57,24 @@ export function generateIsomorphicTests(
                         Enzyme.render(element);
                     });
                 }
+                if (classNameSkipList.includes(componentName)) {
+                    it.skip(`<${componentName}>`);
```

Comment:
use the same test name as line 63.

Reply:
I refactored this so that if a component is on the skipList, both the render and className tests will be skipped. If rendering fails, no use searching for a class